addons/website_forum/populate/mail_message.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo.tools import populate
+
+CP_WEIGHTS = { 1: 35, 2: 30, 3: 25, 4: 10 }
+
+class Message(models.Model):
+    _inherit = "mail.message"
+    _populate_dependencies = ["forum.post"]
+    _populate_sizes = {'small': 100, 'medium': 1000, 'large': 90000 }
+
+    def _populate_factories(self):
+        question_ids = self.env.registry.populated_models['forum.post']
+        answers = self.env['forum.post'].search_read([('parent_id', 'in', question_ids)], ['id'])
+        answer_ids = [id for answer in answers for id in answer.values()]
```

Comment:
`id` is a globally accessible python built-in; although you can shadow it in narrower scopes, it's not good practice.
You can do one of two things :
```suggestion
        answer_ids = [id_ for answer in answers for id_ in answer.values()]
```
or
```suggestion
        answer_ids = [answer_id for answer in answers for answer_id in answer.values()]
```